src/js/analyses/components/Create/QuickAnalyze.tsx

```diff
@@ -139,16 +227,21 @@ export function QuickAnalyze({
                                         onChange={value => setFieldValue("subtractions", value)}
                                     />
 
-                                    <QuickAnalyzeError>{touched.subtractions && errors.subtractions}</QuickAnalyzeError>
+                                    <QuickAnalyzeError>
+                                        {touched.subtractions &&
+                                            (errors.subtractions as ReactNode & (string | string[]))}
+                                    </QuickAnalyzeError>
                                 </>
                             )}
                             <Field
                                 as={IndexSelector}
-                                indexes={compatibleIndexes}
+                                indexes={indexes}
                                 selected={values.indexes}
                                 onChange={value => setFieldValue("indexes", value)}
                             />
-                            <QuickAnalyzeError>{touched.indexes && errors.indexes}</QuickAnalyzeError>{" "}
+                            <QuickAnalyzeError>
+                                {touched.indexes && (errors.indexes as ReactNode & (string | string[]))}
+                            </QuickAnalyzeError>{" "}
```

Comment:
```suggestion
                            </QuickAnalyzeError>
```
Unless it is doing something